ert_shared/ensemble_evaluator/entity/identifiers.py

```diff
@@ -48,6 +48,7 @@
 STAGE_ID = "stage_id"
 START_FILE = "start_file"
 START_TIME = "start_time"
+DURATION = "duration"
```

Comment:
This isn't really a field that is coming as part of the status messages from the ensemble evaluator. I think this "magic" string should not be stored here, as it is only used as a column name in the GUI.

Reply:
I agree, I didn't like it when I added it, it was more of an uncertainty of how to add this.. There should've been a question about this part from me!

Fixed it now though by adding an individually label in the snapshot model instead.